src/main/java/com/endava/internship/collections/Tests.java

```diff
@@ -0,0 +1,31 @@
+package com.endava.internship.collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.*;
+
+import java.time.LocalDate;
+
+public class Tests {
```

Comment:
naming, usually tests are named after the class they are testing, e.g. StudentSetTests

Reply:
done